app/assets/v2/js/pages/new_bounty.js

```diff
@@ -122,139 +125,120 @@ $(document).ready(function(){
-                _alert({ title: "Transaction #1 Submitted", message: msg  }, 'info');
-                next();
+                _alert({ message: "There was an error.  Please try again or contact support." });
+                $('#submitBounty').removeAttr('disabled');
+                return;
             }
-        };
-        var sendERC20Approve = function(){
-            token_contract.approve.estimateGas(bounty_address()
-                ,amount, 
-                function(errors,result){
-                    mixpanel.track("New Bounty Error", {step: 'erc20', error: errors});
-                    var gas = Math.round(erc20_approve_gas * gasMultiplier);
-                    var gasLimit = Math.round(gas * gasLimitMultiplier);
-                    token_contract.approve.sendTransaction(bounty_address()
-                        ,amount, 
-                        {from :account, 
-                            gas:web3.toHex(gas), 
-                            gasLimit: web3.toHex(gasLimit), 
-                            gasPrice:web3.toHex(defaultGasPrice), 
-                        },
-                        erc20_approve_callback);
-                });
-        };
-        // actually send the transactions to web3
-        setTimeout(function(){
-            bounty.bountydetails.call(issueURL, function(error, result){
-                if(error){
-                    console.error(error);
-                    mixpanel.track("New Bounty Error", {step: 'details', error: error});
-                    _alert({ message: "There was an error.  Please try again or contact support." });
-                    unloading_button($('#submitBounty'));
-                    return;
-                }
-                var isOpenAlready = result[4];
-                if(isOpenAlready){
-                    _alert("There is already an open funding on this issue.  Please try again with another issue.");
-                    unloading_button($('#submitBounty'));
-                    return;
-                }
-                if(!isETH){
-                    sendERC20Approve();
-                } else {
-                    sendPostBounty();
-                }
-            });
-        },100);
-        e.preventDefault();
-    });
+            localStorage['dataHash'] = result;  // cache data hash to find bountyId later
+            // bounty is a web3.js eth.contract address
+            // The Ethereum network requires using ether to do stuff on it
+            // issueAndActivateBounty is a method definied in the StandardBounties solidity contract.
+            var bountyIndex = bounty.issueAndActivateBounty(
+                account, 
+                expire_date, 
+                result, 
+                amount, 
+                0x0, 
```

Comment:
@mbeylin i'm so excited we can use arbiters now :)

Reply:
!!! suuuuuper low hanging fruit bounty: write a service that checks when the PR associated with a fulfillment has been accepted, and (while acting as the `arbiter`), accepts the associated submission.

Reply:
@mbeylin yea that would be pretty neat.  I think we should push that out to the next PR though.  This one is already pretty massive.

Reply:
After this is merged I think it makes sense to collect a bunch of additional features that we want and get those in as separate issues/PRs.